core/src/main/scala/io/finch/ToService.scala

```diff
@@ -35,7 +35,17 @@ trait ToService[ES <: HList, CTS <: HList] {
   def apply(
     endpoints: ES,
     includeDateHeader: Boolean,
-    includeServerHeader: Boolean
+    includeServerHeader: Boolean,
+    enableMethodNotAllowed: Boolean = false
+  ): Service[Request, Response] =
+    run(endpoints, includeDateHeader, includeServerHeader, enableMethodNotAllowed, methodNotAllowed = false)
+
+  protected def run(
```

Comment:
What do you think about `applyWithContext` instead? Assuming `methodNotAllowed` is somewhat part of the context that needs to be propagated across the chain of `ToService` instances.

Reply:
Oh, also. How do you feel about moving context-params into a separate param list (to visually distinguish them)?

```scala
def applyWithContext(
  includeDateHeader: Boolean,
  includeServerHeader: Boolean,
  enableMethodNotAllowed: Boolean)(matchedWithoutMethod: Boolean)
```